### PR TITLE
zerotier: add initial package

### PR DESCRIPTION
**Pull request by mwarning**
Maintainer: me
Compile tested: ar71xx, tl-wr703, OpenWrt 15.05
Run tested: ar71xx, tl-wr703, OpenWrt 15.05 (software connects to servers)

Description:
ZeroTier One is a programm to create a global provider-independent virtual private cloud. This project offers OpenWrt/LEDE packages for ZeroTier.

This is the package that has been maintained at this location so far:
https://github.com/mwarning/zerotier-openwrt

### CODE REVIEW

File: net/zerotier/Makefile [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#
```
Comment (diizzyy) [inline net/zerotier/Makefile]: Don't set copyright to OpenWrt unless you're affiliated or under contract.
Set yourself as copyright holder or omit it.

File: net/zerotier/Makefile [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=zerotier
+PKG_VERSION:=1.1.14
+PKG_REV:=ae491c277e6f35d1acbdcbf700e2b834957295ae
+PKG_RELEASE:=4
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=git://github.com/zerotier/ZeroTierOne.git
+PKG_SOURCE_PROTO:=git
```
Comment (diizzyy) [inline net/zerotier/Makefile]: Use release tarball since it's available

File: net/zerotier/patches/0005-use-internal-miniupnpc-and-natpmp [hunk 3]
```diff
@@ -0,0 +1,41 @@
+Index: zerotier-1.1.14/make-linux.mk
```
Comment (diizzyy) [inline net/zerotier/patches/0005-use-internal-miniupnpc-and-natpmp]: This should go, there's a packagesvfor this in the repo already.

**Comment by mwarning** [comment]
@diizzyy all fixed now, hopefully.. :-)

**Comment by bobafetthotmail** [comment]
Please squash all commits into one. In your local git branch write 
**git reset HEAD~5**
to delete the last 5 commits (but leave the modified files intact), then
**git add -i**
to select the files you want to add in your commit, then
**git commit**
and you write the commit message, then
**git push -f**
to send this change to github and in this PR.

**Comment by mwarning** [comment]
I removed some test code that snuck in there. Looks fine now.

**Pull request merged**